src/serverdlg.h

```diff
@@ -36,68 +36,72 @@
 #include <QSystemTrayIcon>
 #include <QSettings>
 #include <QFileDialog>
-#if QT_VERSION >= QT_VERSION_CHECK(5, 6, 0)
-# include <QVersionNumber>
+#if QT_VERSION >= QT_VERSION_CHECK( 5, 6, 0 )
+#include <QVersionNumber>
 #endif
 #include "global.h"
 #include "util.h"
 #include "server.h"
 #include "settings.h"
 #include "ui_serverdlgbase.h"
 
-
 /* Definitions ****************************************************************/
 // update time for GUI controls
-#define GUI_CONTRL_UPDATE_TIME      1000 // ms
+#define GUI_CONTRL_UPDATE_TIME 1000 // ms
 
 // Strings used in multiple places
 #define SREC_NOT_INITIALISED CServerDlg::tr ( "Not initialised" )
-#define SREC_NOT_ENABLED     CServerDlg::tr ( "Not enabled" )
-#define SREC_NOT_RECORDING   CServerDlg::tr ( "Not recording" )
-#define SREC_RECORDING       CServerDlg::tr ( "Recording" )
-
+#define SREC_NOT_ENABLED CServerDlg::tr ( "Not enabled" )
+#define SREC_NOT_RECORDING CServerDlg::tr ( "Not recording" )
+#define SREC_RECORDING CServerDlg::tr ( "Recording" )
 
 /* Classes ********************************************************************/
 class CServerDlg : public CBaseDlg, private Ui_CServerDlgBase
 {
     Q_OBJECT
 
-public:
+  public:
     CServerDlg ( CServer*         pNServP,
                  CServerSettings* pNSetP,
                  const bool       bStartMinimized,
                  QWidget*         parent = nullptr );
 
-protected:
+  protected:
     virtual void changeEvent ( QEvent* pEvent );
     virtual void closeEvent ( QCloseEvent* Event );
 
-    void         UpdateGUIDependencies();
-    void         UpdateSystemTrayIcon ( const bool bIsActive );
-    void         ShowWindowInForeground() { showNormal(); raise(); }
-    void         ModifyAutoStartEntry ( const bool bDoAutoStart );
-    void         UpdateRecorderStatus( QString sessionDir );
+    void UpdateGUIDependencies();
+    void UpdateSystemTrayIcon ( const bool bIsActive );
+    void ShowWindowInForeground()
```

Comment:
OK.  So, we've said "getter/setter-type stuff".  This one's ... yeah, well, borderline.  Probably preferable to split, yes.